packages/flutter_tools/templates/create/pubspec.yaml.tmpl

```diff
@@ -1,6 +1,14 @@
 name: {{projectName}}
 description: {{description}}
 
+# A version number is three numbers separated by dots, like 1.2.43
+# followed by a build number separated by a +.
+# A complete version number looks like 1.2.43+177
+# The build command will use this number to set the version and build number
+# for Android and iOS builds. The run command will ignore it.
```

Comment:
Why will run ignore it?

Reply:
I guess this is deprecated when I'm done with refactoring suggested by mravn-google